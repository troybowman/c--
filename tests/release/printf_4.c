../tree/printf_1.c