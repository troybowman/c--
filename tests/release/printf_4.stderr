error: line 9 - function printf called as part of an expression but does not return a value
error: line 10 - function printf called as part of an expression but does not return a value
error: line 11 - function printf called as part of an expression but does not return a value
error: first argument to printf() must be a string constant, line 11
error: first argument to printf() must be a string constant, line 13
error: first argument to printf() must be a string constant, line 14
error: incompatible argument for format specifier, line 16
error: incompatible argument for format specifier, line 17
error: incompatible argument for format specifier, line 18
error: invalid number of format arguments, line 20
error: invalid number of format arguments, line 22
error: invalid number of format arguments, line 23
c-- returned nonzero exit code